src/components/UniversalTable/UniversalTable.tsx

```diff
@@ -0,0 +1,28 @@
+import {  ReactNode } from 'react';
+import classes from './UniversalTable.module.scss';
+import { UniversalTableProps, BaseData } from './types';
+
+
+const renderCell = (value: any): ReactNode => {
```

Comment:
Why do we have any here?

Reply:
Why does return type only react node if it also can be a string?

Reply:
Now renderCell is returning string type.
![image](https://github.com/BUI-studies/yidlorama/assets/113294189/6f29b8aa-1fb0-4594-9f3b-8cf5101974ea)
Why type 'any'? Because it's the only type that satisfies the condition

Reply:
You should strictly define type of prop. Return type be combined: ```string | (required type)```

Reply:
having type `any` in your code is considered pure safety and bad practice